crates/rpc/rpc/src/eth/sim_bundle.rs

```diff
@@ -335,7 +336,21 @@ where
                     // TODO: since we are looping over iteratively, we are not collecting bundle
                     // logs. We should collect bundle logs when we are processing the bundle items.
                     if logs {
-                        let tx_logs = result.logs().to_vec();
+                        let tx_logs = result
+                            .logs()
+                            .iter()
+                            .enumerate()
+                            .map(|(log_index, log)| alloy_rpc_types_eth::Log {
+                                inner: log.clone(),
+                                block_hash: None,
+                                block_number: None,
+                                block_timestamp: None,
```

Comment:
should those even be set?

I guess block_number and timestamp could be derived from the parent block, but maybe those should all just be None

Reply:
I dug into how the original mev-geth worked, and it actually does something very similiar to the reth implementation of `eth_simulateV1` where it builds a block on top of the chain w/ all the txs in it, and the logs contain the computed blockHash and number for that.  So I think that would be the cleaner longer term fix, but for now its not really necessary.  The txHash & index are the main things that are used to lookup logs in the response, the block number/hash isn't really useful (the number can obviously be derived and the block hash isn't meaningful in this context).